src/internal/packager2/packager2.go

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package packager2 is the new implementation for packager.
+package packager2
```

Comment:
Is the file name intended to be packager2?

Reply:
Yes.